src/main.rs

```diff
@@ -1,6 +1,8 @@
 #![allow(non_snake_case, non_upper_case_globals)]
 #![deny(clippy::option_unwrap_used)]
 
+extern crate glob;
```

Comment:
Ah sorry I missed this one in the previous review. This isn't needed anymore in "rust 2018 edition".

Reply:
It was still failing to build without this, do I need to be running a specific version of `rustc`?

Reply:
Ah that's because you have imported it in `language_server_protocol.rs` like this `use crate::glob::glob`, which imports the `glob` defined in this project's crate, which is not the external dependency but the one you "re-exported" by adding that `export crate glob`. Dropping this line and changing `use crate::glob::glob` to `use glob::glob` in `language_server_protocol.rs` should do it.

Reply:
Looks like that worked, thanks!